Python/initconfig.c

```diff
@@ -789,14 +789,12 @@ _PyConfig_InitCompatConfig(PyConfig *config)
     config->use_frozen_modules = 1;
 #endif
     config->safe_path = 0;
+    /* config_init_int_max_str_digits() sets default limit */
```

Comment:
For me, this comment is not really useful, but you can keep it if it's useful to you ;-) Usually, I just search for "int_max_str_digits" in this C file ;-)